core/templates/pages/story-editor-page/modal-templates/new-chapter-title-modal.component.html

```diff
@@ -17,44 +17,43 @@ <h3>
       </div>
       <input class="form-control protractor-test-new-chapter-title-field oppia-new-chapter-title-field"
              placeholder="Enter the title for this chapter to be displayed to learners in the story viewer page (can be edited later)"
-             ng-model="title" ng-change="resetErrorMsg()" maxlength="<[MAX_CHARS_IN_EXPLORATION_TITLE]>" autofocus>
+             [(ngModel)]="title" [ngModelOptions]="{standalone: true}" (change)="resetErrorMsg()" maxlength="{{ MAX_CHARS_IN_EXPLORATION_TITLE }}" autofocus>
```

Comment:
Why is `standalone: true` needed?

Reply:
It's needed because without this, the 'Create Chapter' button is always in a disabled state.